lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
"Ends up in the viewer" in the sense that viewer doesn't run [any of this stuff](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/templates/report-template.html#L228-L247) itself.

What does end up in viewer is anything we put in report-generator.js. For example, any html/css/js file we read as a string ends up in the browseried bundle. If you look in https://googlechrome.github.io/lighthouse/viewer/src/main.js, you can see html snippets, js file contents, css, data urls. It ends up being lots of stuff.